app/move/controllers/view/view.tabs.urls.js

```diff
@@ -0,0 +1,9 @@
+const getTabsUrls = move => {
+  const tabsUrls = {
+    view: `/move/${move.id}`,
```

Comment:
Could you use relative URLs within the view rather than needing these?

Reply:
So...

```
  const tabsUrls = {
    view: `.../${move.id}`,
    timeline: `timeline`,
  }
```
??

with the actual move link being wrong when it's the current page?

Reply:
Can we have a centralised url thingy already?